python/ray/tests/test_advanced.py

```diff
@@ -749,6 +749,33 @@ def indirect_dep(input):
     assert ray.get(indirect_dep.remote(["hello"])) == "hello"
 
 
+def test_wait_makes_object_local(ray_start_cluster):
+    cluster = ray_start_cluster
+    cluster.add_node(num_cpus=0)
+    cluster.add_node(num_cpus=2)
+    ray.init(address=cluster.address)
+
+    @ray.remote
+    class Foo(object):
+        def method(self):
+            return np.zeros(1024 * 1024)
+
+    a = Foo.remote()
+
+    # Test get makes the object local.
```

Comment:
What is this case testing? How can the object be gotten and not be local?

Reply:
It's a sanity check for the exists check.

Reply:
Can remove.